lib/fetchers/local.rb

```diff
@@ -55,6 +55,10 @@ def archive_path
       @target
     end
 
+    def writable?
+      File.directory?(@target)
```

Comment:
True, in this context by writable? we simply mean "is this the type of source that I actually want to write on disk".  We may still fail with EACCESS or something when we go to write.